main.py

```
@@ -2,47 +2,51 @@
 # Integrates with UMDR MCP stack for full runtime capabilities
 
 from agents.executor import execute_task
-from agents.mutator import mutate_protocol, analyze_all_protocols
+from agents.mutator import mutate_protocol
 from utils.logger import log
 from utils.tracker import get_protocol_stats
 import sys
```

Pylint (3.3.7)

[convention] standard import "sys" should be placed before first party imports "agents.executor.execute_task", "agents.mutator.mutate_protocol", "utils.logger.log", "utils.tracker.get_protocol_stats" 

(C0411)